bundles/admin/admin-layereditor/view/AdminLayerForm.jsx

```diff
@@ -0,0 +1,85 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import {Tabs, TabPane} from '../components/Tabs';
+import {Button} from '../components/Button';
+import {GeneralTabPane} from './AdminLayerForm/GeneralTabPane';
+import {VisualizationTabPane} from './AdminLayerForm/VisualizationTabPane';
+import {AdditionalTabPane} from './AdminLayerForm/AdditionalTabPane';
+import {PermissionsTabPane} from './AdminLayerForm/PermissionsTabPane';
+import {AdminLayerFormService} from './AdminLayerForm/AdminLayerFormService';
+import {StyledRoot} from './AdminLayerForm/AdminLayerFormStyledComponents';
+import {Alert} from '../components/Alert';
+import {GenericContext} from '../../../../src/react/util.jsx';
+export class AdminLayerForm extends React.Component {
+    constructor ({layer, dataProviders, mapLayerGroups, flyout}) {
+        super();
+        this.flyout = flyout;
```

Comment:
We should keep the flyout instance out of the React component. Pass props `flyoutExists` and ´onCancel´ instead.